specification/dimensions/commitmentdiscounttype.md

```diff
@@ -0,0 +1,32 @@
+# Commitment Discount Type
+
+A commitment-based discount is a commitment for an amount of usage or spend throughout a specified term, in exchange for discounted unit pricing on that amount. The commitment may be based on quantities of resource units or monetary value, with various payment options and time frames.
+
+Commitment Discount Type is a provider-assigned name to identify the type of commitment-based discount applied to this record. For example, GCP might refer to a "Committed Use Discount" (CUD) while Azure might call their offering a "Reservation" or "Reserved Instance".
```

Comment:
We don't want to add cloud-specific examples in the spec. That should only be in the supporting content. We could use a fictitious name, but I'm not sure that will convey the point in the same way. Can you see what we do for ServiceName and mimic that? (Sorry, on my phone and can't confirm that right now.)

Reply:
Ok, checked ServiceName and yes, it does not mention it. Moving to supporting content